presto-hive/src/test/java/com/facebook/presto/hive/parquet/AbstractTestParquetReader.java

```diff
@@ -448,17 +776,71 @@ protected Double computeNext(Double previous)
 
     private static ContiguousSet<Integer> intsBetween(int lowerInclusive, int upperExclusive)
     {
-        return ContiguousSet.create(Range.openClosed(lowerInclusive, upperExclusive), DiscreteDomain.integers());
```

Comment:
what is the reason for this change and the same changes below?

Reply:
Parameters of this method are named lowerInclusive/upperExclusive, but the behaviour used to be the opposite. So either openClosed range needed to be changed to closedOpen, or method signature to lowerExclusive/upperInclusive.